README.md

```diff
@@ -65,29 +65,47 @@ on Lattigo:
 
 The current version of Lattigo, (v3.x.x) is fast-evolving and in constant development. Consequently,
 there will still be backward-incompatible changes within this major version, in addition to many bug
-fixes and new features. Hence, we encourage all Lattigo users to update to the latest Lattigo
-version.
- 
+fixes and new features. Hence, we encourage all Lattigo users to update to the latest Lattigo version.
+
 
 See CHANGELOG.md for the current and past versions.
 
+## Stability
+
+To keep a comprehensive history, we prioritize rebases over merges for branches other than `main`.
+Branches with the prefix `dev_` are branches in active development and will be frequently rebased. Hence, we don't recommend depending on them.
+
+## Pull Requests
+
+External pull requests should only be used to propose new functionalities that are substantial and would require a fair amount of work if done on our side. If you plan to open such a pull request, please contact us before doing so to make sure that the proposed changes are
+aligned with our development roadmap.
+
+External pull requests only proposing small or trivial changes will be converted to an issue and closed.
+
+## Support and Issues
+
+The GitHub issues should be only used for bug reports and questions directly related to the use or the implementation of the library.
+Any other issue will be closed, and for this we recommend the use of [GitHub discussions](https://github.com/tuneinsight/lattigo/discussions) or other topic-specific forums instead.
+Any new issue regarding an unexpected behavior of the library or one of its packages must be accompanied 
+by a self-contained `main.go` reproducing the unwanted behavior.
+
 ## License
 
-Lattigo is licensed under the Apache 2.0 License. See LICENSE.
+Lattigo is licensed under the Apache 2.0 License. See [LICENSE](https://github.com/tuneinsight/lattigo/blob/master/LICENSE).
 
 ## Contact
 
-If you want to contribute to Lattigo or you have any suggestion, do not hesitate to contact us at
+If you want to contribute to Lattigo or you have any questions or suggestions, do not hesitate to contact us at
```

Comment:
Shouldn't questions and suggestions go on the public discussion by default ? Should we say "non-public inquires" or something like that ?